doc/index.rst

```diff
@@ -196,6 +196,7 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    :hidden:
 
    code/qml
+   code/qml_allocation
```

Comment:
Where is `qml_allocation.rst` getting added?

Reply:
```suggestion
```